src/test/java/net/logstash/logback/encoder/LogstashAccessEncoderTest.java

```diff
@@ -200,7 +200,7 @@ public void unixTimestampAsNumber() throws Exception {
         
         JsonNode node = MAPPER.readTree(encoded);
         
-        assertThat(node.get("@timestamp").numberValue()).isEqualTo(timestamp);
+        assertThat(node.get("@timestamp").asLong()).isEqualTo(timestamp);
```

Comment:
Per my previous comment, this change will need to be reverted to ensure backwards compatibility.  The json node should be a number value, not a string value that needs to be converted to a Long

Reply:
Sorry, Long is a type of Number? Its just the correct one already.

Reply:
They are not quite the same.

`numberValue()` will return the number if and only if the JsonNode represents a JSON number.  This is the expected condition of this test case.

`asLong()` will convert String JsonNode values to a number.  In this test case, the JsonNode is not expected contain a String value.